frodo.js

```diff
@@ -4,71 +4,71 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-You did a great job handling all the possible error states in the UI. It enhances the user experience significantly.